yellowbrick/text/postag.py

```diff
@@ -167,21 +170,46 @@ def __init__(
         self.colormap = colormap
         self.colors = colors
         self.stack = stack
+        self.parser = parser
+        self.tagger = tagger
+
+    @property
+    def parser(self):
+        return self.__parser
+
+    @parser.setter
+    def parser(self, parser):
+        accepted_parsers = ['nltk', 'spacy']
+        if not parser:
+            self.__parser = None
+        elif parser in accepted_parsers:
+            try:
+                importlib.import_module(parser)
+            except ModuleNotFoundError:
+                raise ModuleNotFoundError("Can't find module '{}' in this environment.".format(parser))
```

Comment:
Great use of `importlib` here! I do like this better than our try/except/None pattern that we've used in other places.

Reply:
Thanks! My main motivation was pep8 compatibility to avoid having the imports below the top of the file.